index.html

```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
                   </ul>
                 </li>
                 <li class="frameworks">
-                  <p class="">Frameworks</p>
+                  <h2> Frameworks </h2>
                 </li>
                 <li class="skills">
                   <h2> Skills </h2>
                 </li>
               </ul>
             </div>
-    </section>
-
-
+          </section>
 
+          <section class="contact__me__section">
+         <div class="contact__me">
+           <h3 class="contact__me__title">Contact Me</h3>
+           <p class="contact__me__text">
+               If you have an application you are interested in developing,
+               a feature that you need built or a project that needs coding.
+               I’d love to help with it
+           </p>
+           <form
+             class="contact__me__form"
+             action="https://formspree.io/f/mjvlknzp"
+             method="post"
+             id="form">
+             <input
```

Comment:
- [ ] Nice working and validating your input fields, kindly add left padding to your input fields to give it space as shown in the Figma template.

![](https://user-images.githubusercontent.com/48371156/165287721-391a1179-e2ed-4a89-8d45-f849c944a2d2.png)